src/components/BoemlyTag/BoemlyTag.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { Tag, TagCloseButton, TagLabel, TagProps } from '@chakra-ui/react';
+
+export interface BoemlyTagProps extends TagProps {
+  isClosable?: boolean;
+  onClose?: () => void;
+}
+
+export const BoemlyTag: React.FC<BoemlyTagProps> = ({
+  isClosable = false,
+  onClose,
+  children,
+  ...props
+}) => {
```

Comment:
```suggestion
export const BoemlyTag = ({
  isClosable = false,
  onClose,
  children,
  ...props
}: BoemlyTagProps): JSX.Element => {
```

You might need to return a React fragment instead of null if you want to display nothing: `<></>`

Reply:
I'll take some time this week to learn the difference 👍